templates/docs/whats-new.md

```diff
@@ -20,7 +20,31 @@ When we add, make significant updates, or deprecate a component we update their
     </tr>
   </thead>
   <tbody>
-  <!-- 3.3.0 -->
+ <!-- 3.4.0 -->
```

Comment:
The deprecated use of p-pagination class name should be listed here as well for future reference. Thanks!